webapp/packages/core-ui/src/Tabs/TabPanel.m.css

```diff
@@ -0,0 +1,10 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+.tabPanel {
+}
```

Comment:
probably we want to have default styles for tabs (underline one) we also can make vertical tabs as a part of base styles (so we will provide only prop to tabs to make them vertical oriented, we will be able to make any styled tabs vertical)

Reply:
fixed
it needed base tab styles (not underline one)
+ there is not tabPanel property for vertical oriented tab panels, so left it as it is